roles/testnode/vars/debian_8.yml

```diff
@@ -0,0 +1,104 @@
+---
+packages:
+  - lsb-release
+  - build-essential
+  - sysstat
+  - gdb
+  - python-configobj
+  - python-gevent
+  - python-dev
+  - python-virtualenv
+  - libevent-dev
+  - fuse
+  - libssl1.0.0
+  - libgoogle-perftools4
+  - cryptsetup-bin
+  - libcrypto++9
+  - iozone3
+  - collectl
+  - nfs-kernel-server
+  # for running ceph
+  - libedit2
+  - xfsprogs
+  - gdisk
+  - parted
+  ###
+  # for setting BIOS settings 
+  - libsmbios-bin
+  ###
+  - libuuid1
+  - libfcgi
+  - btrfs-tools
+  # for compiling helpers and such
+  - libatomic-ops-dev
+  ###
+  # used by workunits
+  - git-core
+  - attr
+  - dbench
+  - bonnie++
+  - valgrind
+  - python-nose
+  - mpich2
+  - libmpich2-dev
+  - ant
+  ###
+  # used by the xfstests tasks
+  - libtool
+  - automake
+  - gettext
+  - uuid-dev
+  - libacl1-dev
+  - bc
+  - xfsdump
+  - dmapi
+  - xfslibs-dev
+  ###
+  # For Mark Nelson
+  - sysprof
+  - pdsh
+  ###
+  # for blktrace and seekwatcher
+  - blktrace
+  - python-numpy
+  - python-matplotlib
+  ###
+  # for qemu
+  - kvm
+  - genisoimage
+  ###
+  # for json_xs to investigate JSON by hand
+  - libjson-xs-perl
+  ###
+  # for pretty-printing xml
+  - xml-twig-tools
+  ###
+  # for java bindings, hadoop, etc.
+  - default-jdk
+  - junit4
+  ###
+  # for disk/etc monitoring
+  - smartmontools
+  - nagios-nrpe-server
+  ###
+  # for samba testing
+  - cifs-utils
+  ###
+  # DistCC for arm
+  - distcc
+  ###
+  # tgt
+  - tgt
+  - open-iscsi
+
+#NOTE: these packages were not found for debian 8, but are present for debian 7
```

Comment:
Anybody have context on these packages?  What we might be using them for or if we still need to be installing them?